job/test/test_job_endpoint.py

```diff
@@ -311,3 +311,11 @@ def test_peak_load_outage_times(self):
         self.assertHttpOK(resp)
         outage_start_time_steps = json.loads(resp.content)["outage_start_time_steps"]
         self.assertEquals(outage_start_time_steps, expected_result)
+    
+    def test_summary_endpoints(self):
+
+        user_uuid = "501d1dd9-9779-470b-a631-01c5fbdee570"
+        resp = self.api_client.get(f'/dev/user/{user_uuid}/summary')
```

Comment:
@rathod-b This is where I thought we needed to use python requests instead of the test api_client because my understanding is that every test uses an temporary database that gets deleted after each test.

Reply:
Honestly, I'm not super worried about creating a test right now. I think the real "test" will be making sure this can be used with the UI which is our main goal right now. I attempted to hook this API branch up to the UI and started a fix a couple of issues on the UI side before I decided to give up and hand it over to Erika to do that. Let's wrap up any known issues and get this branch deployed to the development server so she can start testing it out.